test/playback.test.js

```diff
@@ -12,14 +12,20 @@ let when = function(element, type, cb, condition) {
   });
 };
 
-let playFor = function(player, time, cb) {
-  let targetTime = player.currentTime() + time;
-
-  when(player, 'timeupdate', cb, () => player.currentTime() >= targetTime);
+const playFor = function(player, time, cb) {
```

Comment:
the behavior of the `playFor` method changed when removing `targetTime`, you can no longer call `playFor` several times with different values and expect it to play through those seconds.

Reply:
woops fixed